src/components/resource/OcResource.vue

```diff
@@ -99,6 +100,14 @@ export default {
       return this.isResourceClickable && this.isFolder
     },
 
+    folderLink() {
+      const path = this.resource.path.startsWith("/")
```

Comment:
These three lines could be a one liner with a regexp for stripping the leading slash only if it exists:
`const path = this.resource.path.replace(/^\//, ''); // remove leading slash`

Reply:
Thanks - adjusted :wink: